specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/afdx.json

```diff
@@ -9570,7 +9588,13 @@
       }
     },
     "RuleSetUpdatePropertiesParameters": {
-      "description": "The JSON object that contains the properties of the domain to create."
+      "description": "The JSON object that contains the properties of the domain to create.",
+      "properties": {
+        "order": {
```

Comment:
Can you confirm with Adithya/Megan if order should be a property of the rule set or whether it should be specified where the rule set is referenced by a route? Is the support for RuleSet ordering already available in CP and DP?

Reply:
Yes, priority (order) is defined as a Route property. See my comment here - [https://github.com/ShirleyJ27/azure-rest-api-specs/pull/1#issuecomment-668936724](https://github.com/ShirleyJ27/azure-rest-api-specs/pull/1#issuecomment-668936724)
@megan-beaty do you agree?

---
In reply to: [469961774](https://github.com/ShirleyJ27/azure-rest-api-specs/pull/1#discussion_r469961774) [](ancestors = 469961774)

Reply:
she agreed :)

Reply:
Agree w/ Adithya here. Thanks!

Reply:
@ShirleyJ27 Is the change to make priority (order) a Route property done? I am still seeing it here the latest json too.